schlib/rules/rule3_12.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+from rules.rule import *
+
+class Rule(KLCRule):
+    """
+    Create the methods check and fix to use with the kicad lib files.
+    """
+    def __init__(self, component):
+        super(Rule, self).__init__(component, 'Rule 3.12 - Footprint filters', 'Footprint filters should match all appropriate footprints')
+
+    def checkFilters(self, filters):
+    
+        self.bad_filters = []
+   
+        for filter in filters:
+            errors = []
+            # Filter must contain a "*" wildcard
+            if not '*' in filter:
+                errors.append("Does not contain wildcard ('*') character")
```

Comment:
I think this text will not really help anewby ... better (also minding the comment above):
`This footprint filter should end with an "*".`